core/src/shred_fetch_stage.rs

```diff
@@ -434,170 +425,3 @@ fn check_feature_activation(
         }
     }
 }
-
-#[cfg(test)]
-mod tests {
-    use {
-        super::*,
-        solana_ledger::{
-            blockstore::MAX_DATA_SHREDS_PER_SLOT,
-            shred::{ReedSolomonCache, Shred, ShredFlags},
-        },
-        solana_sdk::packet::Packet,
-    };
-
-    #[test]
-    fn test_data_code_same_index() {
```

Comment:
This test is outdated.
It used to verify that the deduper does not discard shreds with the same index but different types; but the deduper is moved to sigverify.
The test is creating legacy shreds which are now always discarded, so I removed the test.